app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -303,14 +286,20 @@ public void onCreate(Bundle savedInstanceState) {
         if (activeAccount != null) {
             ImageView composeAvatar = findViewById(R.id.composeAvatar);
 
-            if (TextUtils.isEmpty(activeAccount.getProfilePictureUrl())) {
-                composeAvatar.setImageResource(R.drawable.avatar_default);
-            } else {
-                Glide.with(this).load(activeAccount.getProfilePictureUrl())
-                        .error(R.drawable.avatar_default)
-                        .placeholder(R.drawable.avatar_default)
-                        .into(composeAvatar);
-            }
+
+            int[] actionBarSizeAttr = new int[] { R.attr.actionBarSize };
+            TypedArray a = obtainStyledAttributes(null, actionBarSizeAttr);
+            int avatarSize = a.getDimensionPixelSize(0, 1);
+            a.recycle();
+
+            boolean animateAvatars = preferences.getBoolean("animateGifAvatars", false);
```

Comment:
We need some helper which can fetch preference with correct key and correct type. At least we need constants for keys.

Reply:
That's a different thing obv.

Reply:
yes we need that, but not in this PR